lib/clamp-device.in

```diff
@@ -119,22 +119,29 @@ if [ $KMDUMPLLVM == "1" ]; then
   cp $2.promote.bc ./dump.promote.bc
 fi
 
+HIP_HC_IR_FILE=""
+
 # select appropriate ROCm-Device-Libs per AMDGPU_TARGET
 if [ $AMDGPU_TARGET == "kaveri" ]; then
   OCLC_ISA_VERSION_LIB="$ROCM_LIB/oclc_isa_version_700.amdgcn.bc"
+  HIP_HC_IR_FILE=$HIP_HC_IR_GFX700
```

Comment:
I'd prefer not introducing the HIP namespace here and make this more generic.  Could you change it to something like:

HCC_EXTRA_LIBRARIES += " $HCC_EXTRA_LIBRARIES_GFX700"  

Then in HIP, you would specify the arch specific libraries using these HCC env var.  Please apply this change to the other architectures as well